extensions/amp-story/validator-amp-story.protoascii

```diff
@@ -99,6 +99,14 @@ tags: {  # <amp-story>
     name: "title"
     mandatory: true
   }
+  attrs: {
+    name: "live-story"
+    value: ""
+  }
+  attrs: {
+    name: "live-story-disabled"
+    value: ""
+  }
```

Comment:
Sorry, should've seen this during the first review: should we add `live-story-poll-interval` too instead of `data-poll-interval`?

Reply:
Hmm I think `data-poll-interval` is okay to have here, since we will also be using `data-sort-time` and `data-update-time` attributes on the pages. 

I would also rather not move too much from what `live-list` already uses, since it might be harder for us to maintain in the future. And this makes sense semantically in my opinion. WDYT?

Reply:
I think, specifically, data-poll-interval is probably fine.  I don't know if there are other attributes you were intending to use this way as well?

Reply:
No more attributes in the amp-story tag. The other ones can be in the amp-story-page and are:
* data-sort-time
* data-update-time
* data-tombstone

And IMO these make semantic sense as well as they are.